kubetest/objects/ingress.py

```diff
@@ -104,3 +106,33 @@ def is_ready(self) -> bool:
             return False
         else:
             return True
+
+    def has_load_balancer_ingress(self) -> bool:
+        """Check if the ingress has been assigned an ingress.
+
+        Returns:
+            True if an ingress has been assigned; False otherwise.
+        """
+        self.refresh()
+        return self.obj.status.load_balancer.ingress is not None
+
+    def wait_for_load_balancer_ingress(self, timeout: int = None) -> None:
+        """Wait until the ingress has been assigned an ingress.
+
+        Args:
+            timeout: The maximum time to wait in seconds, for the
+            Ingress to be assigned an ingress. If unspecified,
+            this will wait indefinitely. If specified and the timeout
+            is met or exceeded, a TimeoutError will be raised.
+
+        Raises:
+            TimeoutError: The specified timeout was exceeded.
+        """
+        wait_condition = condition.Condition(
+            'Ingress has been assigned an ingress',
+            self.has_load_balancer_endpoint)
```

Comment:
should this be `self.has_load_balancer_ingress`?

Reply:
You are absolutely right!